app/src/components/account/UserMenu.vue

```diff
@@ -139,16 +158,25 @@ export default {
     selectedNetwork: "",
     selectedOption: "",
   }),
+  asyncComputed: {
+    addresses: {
+      cache: false,
```

Comment:
why cache false? can't we use the cache first and then update later? to create a better UX